projects/1.16.1/assets/mekanism/mekanism/lang/zh_cn.json

```
@@ -1,1312 +1,1328 @@
+    "generic.mekanism.with_parenthesis": "%s（%s）",
+    "generic.mekanism.with_two_parenthesis": "%s（%s）（%s）",
+    "gui.mekanism.chemical": "化学品：%s",
+    "gui.mekanism.chemical_dissolution_chamber.short": "化学溶解室",
+    "gui.mekanism.chemical_infuser.short": "化学灌注器",
+    "gui.mekanism.close": "关闭",
+    "gui.mekanism.color_picker": "取色器",
+    "gui.mekanism.compass": "指南针",
+    "gui.mekanism.danger": "危险",
+    "gui.mekanism.default": "默认",
+    "gui.mekanism.delay": "延迟：%st",
+    "gui.mekanism.dissipated": "散失：%s/t",
+    "gui.mekanism.dynamic_tank": "动态储罐",
+    "gui.mekanism.eject": "弹出：%s",
+    "gui.mekanism.empty": "空",
+    "gui.mekanism.energy": "能量：%s",
+    "gui.mekanism.false": "否",
+    "gui.mekanism.finished": "完成：%s",
+    "gui.mekanism.fuel": "燃料：%s",
+    "gui.mekanism.gas": "气体：%s",
+    "gui.mekanism.height": "高度",
+    "gui.mekanism.helmet_options": "头盔选项",
+    "gui.mekanism.hud_overlay": "HUD覆盖：",
+    "gui.mekanism.infinite": "无限",
+    "gui.mekanism.infuse_type": "灌注类型：%s",
+    "gui.mekanism.liquid": "液体：%s",
+    "gui.mekanism.max": "最高：%s",
+    "gui.mekanism.max_output": "最大输出：%s/t",
+    "gui.mekanism.min": "最低：%s",
+    "gui.mekanism.move_down": "下移",
+    "gui.mekanism.move_up": "上移",
+    "gui.mekanism.needed": "需要：%s",
+    "gui.mekanism.needed_per_tick": "需要：%s/t",
+    "gui.mekanism.no_delay": "没有延迟",
+    "gui.mekanism.no_fluid": "没有流体",
+    "gui.mekanism.no_recipe": "（没有配方）",
+    "gui.mekanism.none": "无",
+    "gui.mekanism.opacity": "不透明度",
+    "gui.mekanism.pigment": "颜料：%s",
+    "gui.mekanism.process_rate": "处理速率：%s",
+    "gui.mekanism.process_rate_mb": "处理速率：%s mB/t",
+    "gui.mekanism.progress": "进度：%s",
+    "gui.mekanism.radiation_dose": "辐射剂量：%s",
+    "gui.mekanism.radiation_exposure": "辐射暴露：%s",
+    "gui.mekanism.rgb": "三原色：",
+    "gui.mekanism.set": "设置:",
+    "gui.mekanism.slurry": "泥浆：%s",
+    "gui.mekanism.storing": "存储：%s",
+    "gui.mekanism.ticks_required": "所需 tick：%s",
+    "gui.mekanism.transferred_rate": "已传输：%s/t",
+    "gui.mekanism.true": "是",
+    "gui.mekanism.unit": "单元：%s",
+    "gui.mekanism.upgrade": "%s 升级",
+    "gui.mekanism.upgrades": "升级",
+    "gui.mekanism.upgrades.amount": "数量：%s/%s",
+    "gui.mekanism.upgrades.effect": "效率：%sx",
+    "gui.mekanism.upgrades.no_selection": "没有选中.",
+    "gui.mekanism.upgrades.supported": "支持:",
+    "gui.mekanism.usage": "用法：%s/t",
+    "gui.mekanism.using": "能耗：%s/t",
+    "gui.mekanism.volume": "容积：%s",
+    "gui.mekanism.warning": "警告",
+    "gui.mekanism.width": "宽度",
+    "holiday.mekanism.border": "%s%s%1$s",
+    "holiday.mekanism.christmas.1": "圣诞快乐, %s!",
+    "holiday.mekanism.christmas.2": "祝你圣诞快乐",
+    "holiday.mekanism.christmas.3": "和你的家庭一起度过一个",
+    "holiday.mekanism.christmas.4": "轻松的假期 :)",
+    "holiday.mekanism.may_4.1": "愿原力与你同在，%s！",
+    "holiday.mekanism.new_year.1": "新年快乐, %s!",
+    "holiday.mekanism.new_year.2": "在我们进入这个激动人心的",
+    "holiday.mekanism.new_year.3": "新的 %s 年之际，向您致以最良好的祝愿 :)",
```

|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
    "holiday.mekanism.christmas.1": "圣诞快乐，%s！",
    "holiday.mekanism.christmas.2": "祝你圣诞快乐",
    "holiday.mekanism.christmas.3": "愿你能和你的家庭一起度过一个",
    "holiday.mekanism.christmas.4": "轻松的假期 :)",
    "holiday.mekanism.may_4.1": "愿原力与你同在，%s！",
    "holiday.mekanism.new_year.1": "新年快乐，%s！",
    "holiday.mekanism.new_year.2": "在我们进入激动人心的",
    "holiday.mekanism.new_year.3": "%s 年新年之际，向你致以最好的祝愿 :)",
```